src/main/java/ch/uzh/ifi/hase/soprafs21/entities/ActivityPreset.java

```diff
@@ -0,0 +1,56 @@
+package ch.uzh.ifi.hase.soprafs21.entities;
+
+import ch.uzh.ifi.hase.soprafs21.constant.ActivityCategory;
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+/**
+ * This class is used to represent the activities that are used in the UserActivity
+ */
+@Entity
+@Table(name = "ActivityPreset")
+public class ActivityPreset implements Serializable{
+
+    private static final long serialVersionUID = 1L;
+
+    @Id
+    @GeneratedValue
+    private Long id;
+
+    @Column(nullable = false, unique = true)
+    private String activityName;
+
+    @Column(nullable = false)
+    private ActivityCategory activityCategory;
+
+    /**
+     * googlePOI was left out, since we have to discuss how the relationship shall be between
+     * the GooglePOI object and the activityPreset --> should one activity have multiple locations, since for example football can be played on multiple locations?
+     private GooglePOI googlePOI;
+     */
```

Comment:
```suggestion
     private GooglePOI googlePOI;
```

I'd say that's alright. The GooglePOI object only contains minimal information for the API request (e.g. 'keyword = sushi' for an activity _eat at sushi restaurant_). There is no location saved in `GooglePOI`.